### ISSUE TITLE
"purescript without node" should use psc-package

### ISSUE THREAD

**Issue by matthewleon** [purescript/documentation#109]
https://github.com/purescript/documentation/blob/master/guides/PureScript-Without-Node.md

Not sure how others feel about this, but to me it simplifies Node-free development.

**Comment by hdgarrood** [comment]
Yes, definitely. That page is only like that because it predates psc-package.

**Comment by chexxor** [comment]
I'd prefer to decouple guides on dependency management tools and bundling tools from that "purescript without node" article. Would need to create those guides first, then can simplify that article. I imagine the simplified article would be like "Use tool X for dep management. See guide X. Use tool Y for bundling. See guide Y."

**Comment by hdgarrood** [comment]
If the purpose of the guide is to present a sensible workflow for purescript without node, then psc-package is definitely the right choice. We could certainly do with more docs around the compiler CLI, though, which discuss only the `purs` executable and don’t bring in other tools, but I think that’s a different thing.

**Comment by toraritte** [comment]
Would it be possible adding a demonstration on how to use PureScript with other JavaScript backends (e.g., [deno](https://deno.land/))?

**Comment by nwolverson** [comment]
This could now instead/also refer to spago, I think the only node dependency is a runtime on for node-dependant commands (like run and test)

**Comment by thomashoneyman** [comment]
I don't think there's interest among the core team in maintaining `psc-package`, so I agree this should refer to Spago.
